apps/documentation/src/routes/documentation/styles/tailwind-css.mdx

```diff
@@ -0,0 +1,55 @@
+import MetaTags from '@/components/MetaTags'
+
+<MetaTags
+  title="Tuono - Tailwind CSS"
+  canonical="https://tuono.dev/documentation/styles/css-modules"
+/>
+
+import Breadcrumbs from '@/components/Breadcrumbs'
+
+<Breadcrumbs breadcrumbs={[{ label: 'Tailwind CSS' }]} />
+
+# Tailwind CSS
+
+Tailwind CSS is a popular utility-first CSS framework.
+
+## Installing Tailwind
```

Comment:
Instead of using heading, I would prefer a numbered list since it's a setup guide.
Something similar can be found on: https://tuono.dev/documentation/contributing/local-development#tuono-development